e2e/wdio/core/tests/table.e2e-spec.ts

```diff
@@ -0,0 +1,232 @@
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(tableExample);
+    }, 1);
+
+    it('should check clickability links for all examples', () => {
+        for (let i = 0; i < 11; i++) {
+            checkIsLinkClickable(componentExampleArr[i]);
+        }
+    });
+
+    describe('Check Table with Toolbar example', function() {
+
+        it('should check display busy indicator', () => {
+            scrollIntoView(tableToolbarExample);
+            click(tableToolbarExample + button, 1);
+            expect(isElementDisplayed(busyIndicator)).toBe(true, 'busy indicator not displayed');
+        });
+
+        it('should check adding a new item', () => {
+            scrollIntoView(tableToolbarExample);
+            click(tableToolbarExample + button, 2);
+            for (let i = 0; i < 3; i++) {
+                setValue(dialogContent + inputField, 'test', i);
+            }
+            click(dialogContent + button, 1);
+            saveElementScreenshot(tableToolbarExample + table, 'table-toolbar-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder());
+            expect(checkElementScreenshot(tableToolbarExample + table, 'table-toolbar-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder())).toBeLessThan(5, `element item state mismatch`);
+        });
+
+        it('should check searching table item', () => {
+            scrollIntoView(tableToolbarExample);
+            setValue(tableToolbarExample + inputField, 'Kale');
+            const rowLength = getElementArrayLength(tableToolbarExample + tableRow);
+            expect(rowLength).toEqual(1);
+            const cellLength = getElementArrayLength(tableToolbarExample + tableRow + tableCell);
+            for (let i = 0; i < cellLength; i++) {
+                expect(getText(tableToolbarExample + tableRow + tableCell, i)).toBe(tableCellArr[i]);
+            }
+        });
+    });
+
+    describe('Check Customizable Columns example', function() {
+
+        it('should check that table customization work correctly', () => {
+            scrollIntoView(tableCustomColumnsExample);
+            click(tableCustomColumnsExample + button);
+            click(dialogContent + button, 1);
+            click(dialogContent + button, 2);
+            saveElementScreenshot(tableCustomColumnsExample + table, 'table-custom-columns-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder());
+            expect(checkElementScreenshot(tableCustomColumnsExample + table, 'table-custom-columns-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder())).toBeLessThan(5, `element item state mismatch`);
+        });
+
+        it('should check search work correctly', () => {
+            scrollIntoView(tableCustomColumnsExample);
+            click(tableCustomColumnsExample + button);
+            setValue(dialogContent + inputGroup, 'date');
```

Comment:
use your test variable dateTestText instead just string 'date', will be easier to read code wthout needed to find this variable in fixtures